### PR TITLE
[Refactor] Volume Onboarding

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_volume_onboarding.go [hunk 1]
```diff
@@ -39,11 +39,11 @@ func ResourceIBMPIVolumeOnboarding() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 
-			helpers.PICloudInstanceId: {
-				Type:        schema.TypeString,
-				Required:    true,
-				ForceNew:    true,
+			Arg_CloudInstanceID: {
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_volume_onboarding.go:42]:
```suggestion
// Arguments
			Arg_CloudInstanceID: {
```

File: ibm/service/power/resource_ibm_pi_volume_onboarding.go [hunk 2]
```diff
@@ -39,11 +39,11 @@ func ResourceIBMPIVolumeOnboarding() *schema.Resource {
...
 				Description: "Cloud Instance ID - This is the service_instance_id.",
+				ForceNew:    true,
+				Required:    true,
+				Type:        schema.TypeString,
 			},
 
 			piOnboardingVolumes: {
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_volume_onboarding.go:49]: All of these should be moved to the constant file and rename with `Arg_` prefix
eg:
`Arg_OnboardingVolumes = "pi_onboarding_volumes"`
```
const (
	piOnboardingVolumes   = "pi_onboarding_volumes"
	piAuxiliaryVolumes    = "pi_auxiliary_volumes"
	piAuxiliaryVolumeName = "pi_auxiliary_volume_name"
	piSourceCRN           = "pi_source_crn"
	piDisplayName         = "pi_display_name"
	piDescription         = "pi_description"
)
```

**Review by michaelkad — CHANGES_REQUESTED** [review]

**Comment by ismirlia** [comment]
```
--- PASS: TestAccIBMPIVolumeOnboardingbasic (16.77s)
PASS
```

**Comment by ismirlia** [comment]
Opened: https://github.com/powervs-ibm/terraform-provider-ibm/pull/168 closing this

**Pull request closed without merging**